client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-page-preview.tsx

```diff
@@ -26,6 +26,17 @@ interface PatternPagePreviewProps {
 const PATTERN_PAGE_PREVIEW_ITEM_VIEWPORT_HEIGHT = 500;
 const PATTERN_PAGE_PREVIEW_ITEM_VIEWPORT_WIDTH = 1080;
 
+// A copy of the title block in Creatio 2's page.html.
+const getPageTitlePattern = ( title: string ) => `
+	<div
+		class="wp-block-group has-global-padding is-layout-constrained wp-container-9 wp-block-group-is-layout-constrained"
```

Comment:
```suggestion
		class="wp-block-group has-global-padding is-layout-constrained wp-block-group-is-layout-constrained"
```

I believe `wp-container-9` is an autogenerated class, which we can remove.

Reply:
Good catch!